### DISCUSSION TITLE
Merged and Pushed Development Branch

### DISCUSSION THREAD

**Discussion by DerpSnepbek** [DerpSnepbek/Tafe24S1T112_Ext#6] [General]
Okay, so i've just merged the branches together and pushed to the remote development branch, and it looks like this when you clone it...
![image](https://github.com/DerpSnepbek/Tafe24S1T112_Ext/assets/161171900/51c986ad-640b-4777-9f29-7a8157ae11c9)

I believe what we have done is correct, because if you look at some examples from the session 6 exercises (where there was a task similar to what we're doing now), the revision graph looks similar to what we've got now.
![image](https://github.com/DerpSnepbek/Tafe24S1T112_Ext/assets/161171900/cf607677-d621-400c-8cc6-2db82958f40f)

So i think we should be okay to continue without re-starting hahah. Let me know what you think?

**Comment by Sunnyyu33a** [comment]
Good! Let's go with it
